ospf6d/ospf6d.h

```diff
@@ -23,6 +23,7 @@
 
 #include "libospf.h"
 #include "thread.h"
+#include "lib/json.h"
```

Comment:
Please put this in the appropriate file only.

Reply:
resolved